docs/release_notes.md

```diff
@@ -1,5 +1,52 @@
 # Release Notes (What's New)
 
+## Version 2.0.0 (Upcoming Release)
+
+For a list of all changes in this release, see the [full changelog](https://github.com/nci/scores/compare/1.3.0...develop). Below are the changes we think users may wish to be aware of.
+
+### Breaking Changes
+
+- The function `scores.probability.tw_crps_for_ensemble` previously took an optional (mis-spelled) argument `chainging_func_kwargs`. The spelling has been corrected and the argument is now `chaining_func_kwargs`.
+- For developers, the version of the `black` and `mypy` tools have been updated. As such, developers will need to ensure they re-install the `scores` package with `pip install -e .[develop]` to get the correct versions of these tools.
```

Comment:
I would suggest changing "For developers," to something like "For those who develop on `scores`," or something along those lines.

Reply:
Or maybe "For `scores` developers," might be better?

Reply:
Wording updated following offline discussion